### PR TITLE
Fixes part of #45: Add separate meta-tags for components.

### LINKED ISSUE #45: Final Touchup
Following is the final worklist to get the site ready for the public reveal

- [x] Image Webp
- [x] Image sizes and lazy loading
- [x] accessible tags
- [x] previewer additon
- [x] project update
- [x] dns change
- [x] alums addition
- [x] members check
- [x] carousel errors fix
- [x] carousel text overflow
- [x] Meta tags
- [x] Image Selection at different places

### PR DESCRIPTION
**Pull request by Manan-Rathi**
## Description

Add in meta tags and description.

## Fixes

Fixes #45  by @Manan-Rathi 

## Types of changes

- [ ] Bug/Hot fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring Old Code ( non-breaking change replacing already existing code )
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** guidelines in repository.
- [x] My pull request has a descriptive title and is attached to an existing issue.
- [x] I haven't repeated the code and it follows established code style of repository.
- [x] My branch is up-to-date with upstream branch.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Manan-Rathi** [comment]
PTAL @king-11

File: nuxt.config.ts [hunk 1]
```diff
@@ -16,9 +16,41 @@ export default {
...
+      },
+      {
+        hid: 'og:description',
+        property: 'og:description',
+        content: 'We at Club of Programmers Software Development Group are a bunch of designers and developers who aim to encourage the development of technology and innovation in IIT BHU campus (and beyond) by learning, sharing knowledge and solving problems.'
+      },
+      {
+        hid: 'og:image',
+        property: 'og:image',
+        content: 'static/COPS-Logo.webp'
```
Comment (king-11) [inline nuxt.config.ts:34]: this won't work instead just post the image on some discord server and copy its link here

File: nuxt.config.ts [hunk 2]
```diff
@@ -16,9 +16,41 @@ export default {
...
+      },
+      {
+        hid: 'twitter:description',
+        name: 'twitter:description',
+        content: 'We at Club of Programmers Software Development Group are a bunch of designers and developers who aim to encourage the development of technology and innovation in IIT BHU campus (and beyond) by learning, sharing knowledge and solving problems.'
+      },
+      {
+        hid: 'twitter:image',
+        name: 'twitter:image',
+        content: 'static/COPS-Logo.webp'
```
Comment (king-11) [inline nuxt.config.ts:49]: similarly this too

**Review by king-11 — CHANGES_REQUESTED** [review]

File: nuxt.config.ts (hunk 2 above)
Comment (Manan-Rathi) [reply nuxt.config.ts]: Done!

**Comment by Manan-Rathi** [comment]
PTAL @king-11. Changed to cdn links.

**Review by king-11 — APPROVED** [review]

**Comment by Manan-Rathi** [comment]
Pulled latest changes from main.

**Review by king-11 — APPROVED** [review]

**Pull request merged**